changelog.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 0.3.10
+
+Creates new Debian package `securedrop-keyring` for managing the SecureDrop
+Release Signing Key. Rotates the signing key currently in use by setting
+a dependency on the other Debian packages, so that currently running deployments
+will have their apt keyrings updated via automatic nightly updates.
```

Comment:
This wording is confusing. We set the dependency to ensure that the securedrop-keyring package will be installed automatically through cron-apt on existing installations. We are also rotating the signing key, but that is an orthogonal concern.